sites/public/src/components/listings/search/ListingsSearchCombined.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react"
+import { ListingSearchParams, generateSearchQuery } from "../../../lib/listings/search"
+import { ListingService } from "../../../lib/listings/listing-service"
+import { ListingsCombined } from "../ListingsCombined"
+import { AppearanceBorderType, AppearanceSizeType, Button } from "@bloom-housing/ui-components"
+import { FormOption, ListingsSearchModal } from "./ListingsSearchModal"
+
+type ListingsSearchCombinedProps = {
+  searchString?: string
+  googleMapsApiKey: string
+  listingsEndpoint: string
+  bedrooms: FormOption[]
+  bathrooms: FormOption[]
+  counties: FormOption[]
+}
+
+/**
+ * This combines the search form with the listings map/list. Listings are updated
+ * in both when the search form is submitted.
+ *
+ * @param props
+ * @returns
+ */
+export function ListingsSearchCombined(props: ListingsSearchCombinedProps) {
+  const [listings, setListings] = useState([])
+  const [modalOpen, setModalOpen] = useState(false)
+  const [filterCount, setFilterCount] = useState(0)
+
+  const onFormSubmit = async (params: ListingSearchParams) => {
+    const qb = generateSearchQuery(params)
+    const listingService = new ListingService(props.listingsEndpoint)
+    const result = await listingService.searchListings(qb)
+
+    const listings = result.items
+    const meta = result.meta
+
+    // Keeping this until pagination is implemented
+    console.log(
+      `Showing ${meta.itemCount} listings of ${meta.totalItems} total (page ${meta.currentPage} of ${meta.totalPages})`
+    )
+
+    setListings(listings)
+  }
+
+  const onModalClose = () => {
+    setModalOpen(false)
+  }
+
+  const updateFilterCount = (count: number) => {
+    setFilterCount(count)
+  }
+
+  return (
+    <div>
+      <div style={{ width: "100%", display: "flex" }}>
+        <div style={{ flexGrow: 1 }}></div>
+        <Button
+          border={AppearanceBorderType.borderless}
+          size={AppearanceSizeType.small}
+          onClick={() => {
+            setModalOpen(true)
+          }}
+        >
+          {`Filters ${filterCount}`}
```

Comment:
Can you add this to general.json and add translations, or add a comment/track it somewhere that translations are needed for this?

Reply:
Also, I understand that the styling will be done in future work so the style above can be changed later to fit the mocks. However I see that the mocks also have the number of listings so not just "Filters 3" but "Filters 3" and then "+99 listings in Bay Area". Is this something that you can add in this PR so we also show the length of props.listings somewhere?

Reply:
If needed, I'm willing to volunteer to do the translations, if you just want to add it to general.json, @chriscasto.

@ifranch I had told Chris to not do the +99 listings in the Bay Area section and that we'd cover it in subsequent PR, but @chriscasto if it's easy for you to do, I can/will do the translations for it too, if you add it here.

Reply:
The listing count can't be updated on filter change without significant effort.  It can be updated on submission, but I think that might be confusing if the number of filters changes when the user adjusts search criteria but the number of results does not.  Should we have UX weigh in on this?

@phattya I can add that in.  Do we have a naming convention we're using since this is net new?

Reply:
Never mind, I went ahead and made the change.  Please rename as necessary